packages/rate-limit/README.md

```diff
@@ -0,0 +1,50 @@
+| **statusCode**             |                              <code>number</code>                              |                    249                     | HTTP status code returned when max is exceeded.                                                                                                                                                                                                                          |
+| **skipFailedRequests**     |                             <code>boolean</code>                              |                   false                    | When set to true, failed requests won't be counted.                                                                                                                                                                                                                      |
+| **skipSuccessfulRequests** |                             <code>boolean</code>                              |                   false                    | When set to true successful requests (response status < 400) won't be counted.                                                                                                                                                                                           |
+| **keyGenerator**           |             <code>(req: Request, res: Response) => string</code>              |              (req) => req.ip               | Function used to generate keys.                                                                                                                                                                                                                                          |
+| **shouldSkip**             |             <code>(req: Request, res: Response) => boolean</code>             |                () => false                 | Determine per request if it should be skipped by the middleware                                                                                                                                                                                                          |
+| **onLimitReached**         |      <code>onLimitReached: (req: Request, res: Response) => void</code>       |                  () => {}                  | Function that is called the first time a user hits the rate limit within a given window.                                                                                                                                                                                 |
+| **store**                  |                              <code>Store</code>                               |                MemoryStore                 | By default a MemoryStore is used. [Rate Limit Redis](https://www.npmjs.com/package/rate-limit-redis), [Rate Limit Memcached](https://www.npmjs.com/package/rate-limit-memcached) and [Rate Limit Mongo](https://www.npmjs.com/package/rate-limit-mongo) can be used too. |
+
+## License
+
```

Comment:
Please assign license not to me but yourself because you wrote the module

also, please add a section "Alternatives" with 2-3 similar packages for Express / http